client/src/components/GroceriesList.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react";
+import { getGroceries } from "../utils/api";
+
+export default function GroceriesList() {
+  const [grocery, setGrocery] = useState(null);
+
+  useEffect(() => {
+    async function doGet() {
+      const grocery = await getGroceries();
+      setGrocery(grocery);
+      console.log(grocery);
+      console.log(grocery.title, grocery.name);
```

Comment:
don't leave them there before uploading it 😉 or maybe make a comment at least, why they have to be there
```suggestion
```

Reply:
Currently they show me that my `GroceriesList` function is working till that particular point. 
Output is:
```(4) [{…}, {…}, {…}, {…}]
0: {title: "fruits", name: Array(3)}
1: {title: "drinks", name: Array(5)}
2: {title: "vegetables", name: Array(7)}
3: {title: "cereals", name: Array(2)}
length: 4
__proto__: Array(0)
```
```
undefined undefined
```

Reply:
okay line 11 is the first one?

if so, then grocery isnt an object, it is an array, so line 12 cant work